src/DataService/Media/AvailabilityWindow.php

```diff
@@ -0,0 +1,90 @@
+<?php
```

Comment:
All of these next classes are generated code. It would be OK to skip them and go to just the test classes below.